CODE/explain_yeast_resps.py

```diff
@@ -37,8 +37,6 @@ def parse_args(argv):
     parser.add_argument(
         '--model_config', default='MODEL_CONFIG/yeast_default_config.json',
         help='Json file for pretrained model hyperparameters.')
-    parser.add_argument(
-        '--disable_shap', action='store_true',)
```

Comment:
Are we not adding a flag for the yeast scripts for enabling SHAP value calculations like we did for human?

Reply:
I don't intend to offer that option. Disabling shap is only meant for running permutation efficiently.